programs/bpf_loader/src/syscalls/mod.rs

```diff
@@ -226,6 +228,33 @@ impl HasherImpl for Keccak256Hasher {
     }
 }
 
```

Comment:
I'd add an explanation about this struct and why it's needed so it's clearer to the maintainers.